swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -17,6 +17,8 @@ enum PacketTunnelProviderError: Error {
 
 class PacketTunnelProvider: NEPacketTunnelProvider {
   private var adapter: Adapter?
+  private var hasReceivedFirstConfig = false
```

Comment:
Do we still need this?

Reply:
thanks for spotting, why swift doesn't spot unused variables is just beyond me 🤷‍♂️ 

maybe it's fixed in new Swift or some other tooling.